.github/workflows/cmake.yml

```diff
@@ -116,6 +116,10 @@ jobs:
       run: cmake --build build --config ${{ env.cmake_build_type }} --parallel 2
     - name: install
       run: cmake --install build --config ${{ env.cmake_build_type }}
+    - name: test
+      run: |
+        cmake -S ./test/test-use-cmake -B build-test -DCMAKE_PREFIX_PATH="${{ matrix.install_dir }}"
```

Comment:
We normally use underscores instead of dashes in file and folder names.

Reply:
wops okay, will fix

Reply:
Thanks. Underscores now used.